src/Apcu/bootstrap.php

```diff
@@ -15,37 +15,41 @@
     return;
 }
 
+if (PHP_VERSION_ID >= 80000) {
+    return require __DIR__.'/bootstrap.php8';
```

Comment:
Encoding the PHP version in the file suffix went out of fashion since the good old php3 days. This file extension is a bit surprising and has already caused interesting side effects (FriendsOfPHP/PHP-CS-Fixer#5324). I would appreciate if we would find a better naming convention and use the standard `.php` file extension for PHP code.

Reply:
updated